master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -151,107 +111,153 @@ spec:
 EOF
 ```
 
-It now shows up as a [policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy) object in the Calico API.
+#### Verify Access - Allowed Nginx Ingress
+
+Now ingress traffic to nginx will be allowed.  We can see that this is the case by switching over to our "access" pod in the namespace and attempting to access the nginx service.
 
 ```shell
-$ calicoctl get policy -o wide
-NAME                                ORDER   SELECTOR
-advanced-policy-demo.access-nginx   1000    calico/k8s_ns == 'advanced-policy-demo' && run == 'nginx'
-advanced-policy-demo.default-deny   1000    calico/k8s_ns == 'advanced-policy-demo'
+/ # wget -q --timeout=5 nginx -O -
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>...
 ```
 
-After creating the policy, we can now access the nginx Service.  We also see that the pod can
-access google.com on the public internet.  This is because we have not defined any egress policy.
+After creating the policy, we can now access the nginx Service.
 
-```
-$ kubectl run --namespace=advanced-policy-demo access --rm -ti --image busybox /bin/sh
-Waiting for pod advanced-policy-demo/access-472357175-y0m47 to be running, status is Pending, pod ready: false
+### 4. Deny all egress traffic
 
-If you don't see a command prompt, try pressing enter.
+Enable egress isolation on the namespace by deploying a [default deny all egress traffic policy](https://Kubernetes.io/docs/concepts/services-networking/network-policies/#4-deny-all-egress-traffic).
 
-/ # wget -q --timeout=5 nginx -O -
-...
-/ # ping google.com
-PING google.com (216.58.219.206): 56 data bytes
-64 bytes from 216.58.219.206: seq=0 ttl=61 time=14.365 ms
+```shell
+kubectl create -f - <<EOF
+apiVersion: networking.Kubernetes.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-egress
+  namespace: advanced-policy-demo
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+EOF
 ```
 
-### Prevent outgoing connections from pods
+#### Verify Access - Denied All Egress
 
-Kubernetes NetworkPolicy does not provide a way to prevent outgoing connections from pods.  However,
-Calico does. In this section we'll create egress policies using `calicoctl`
-that allow the outgoing connections in the `advanced-policy-demo` namespace we
-want; all other egress traffic will be denied.
+Now any ingress or egress traffic which is not explicitly allowed by a policy will be denied.
 
-To do this, we'll need to create a policy which selects all pods in the namespace, and allows
-egress traffic to other pods in the namespace.
+We can see that this is the case by switching over to our "access" pod in the
+namespace and attempting to lookup nginx or access google.com.
 
+```shell
+/ # nslookup nginx
+Server:    10.96.0.10
+Address 1: 10.96.0.10
+
+
+nslookup: can't resolve 'nginx'
+/ # wget -q --timeout=5 google.com -O -
```

Comment:
I think the second command here should also be `nslookup google.com`

Reply:
I intentionally set it to wget just to illustrate that DNS and HTTP traffic are both blocked (although I realize google.com technically never is able to do DNS in order to do HTTP).  I'm open to change it if you prefer.  I added some clarity to the previous wording.